tests/test_core.py

```diff
@@ -457,6 +457,7 @@ def test_normalize_query():
         (2, False),
         ('foo', False),
         ('foo\\*bar', False),
```

Comment:
Is this correct?  `'foo\\*bar'` should reduce to `'foo*bar'` ...right?  Because `\\` is an escaped `\`?  Which would leave the `*`?

Reply:
As an update, this turned out to be harder than I anticipated. 

> Is this correct? 'foo\\*bar' should reduce to 'foo*bar' ...right?

It appears that the `\\` and `\` are being treated the same:

```python
In [40]: value = 'foo\\*bar'                                                                      

In [41]: char = "*"                                                                               

In [42]: value.replace(f'\{char}', '')                                                            
Out[42]: 'foobar'

In [43]: value = 'foo\*bar'                                                                       

In [44]: value.replace(f'\{char}', '')                                                            
Out[44]: 'foobar'

```

I am still looking into this, but I feel like I am starting to get confused ;)

Reply:
Is this a Python thing or a `re` thing?  What about something like `r'\\*'`?

Reply:
I was wrong.  You are correct.  Your original tests should be fine.